test-plans/README.md

```diff
@@ -0,0 +1,33 @@
+# test plans
+
+Contains all the test-plans of all our services, and the test plan templates.
```

Comment:
```suggestion
Contains all test plans of all our services, and the templates (warning: files inside this folder are in German).
```

Reply:
should we do it in german? we should discuss this in the daily

Reply:
As discussed:
Luckily, the majority is for English, so our test plans will also be in English.